addon/ng2/blueprints/component/files/__path__/__name__.component.spec.ts

```diff
@@ -1,24 +1,25 @@
 import {
-  it,
-  iit,
+  beforeEachProviders,
   describe,
   ddescribe,
   expect,
+  iit,
+  it,
   inject,
   injectAsync,
-  TestComponentBuilder,
-  beforeEachProviders
+  ComponentFixture,
+  TestComponentBuilder
 } from 'angular2/testing';
 import {provide} from 'angular2/core';
-import {<%= classifiedModuleName %>} from './<%= dasherizedModuleName %>';
+import {<%= classifiedModuleName %>Component} from './<%= dasherizedModuleName %>.component';
 
 describe('<%= classifiedModuleName %> Component', () => {
 
   beforeEachProviders((): any[] => []);
 
 
   it('should ...', injectAsync([TestComponentBuilder], (tcb:TestComponentBuilder) => {
-    return tcb.createAsync(<%= classifiedModuleName %>).then((fixture) => {
+    return tcb.createAsync(<%= classifiedModuleName %>Component).then((fixture: ComponentFixture) => {
```

Comment:
Is `<%= classifiedModuleName %>Component` better than adding `Component` to the `moduleName` instead? Just curious.

We could even do logic to see if the user is saying `ng g component my-comp.component` we don't add a second component to it. WDYT?